ethcore/light/src/net/load_timer.rs

```diff
@@ -199,15 +199,15 @@ pub struct FileStore(pub PathBuf);
 impl SampleStore for FileStore {
 	fn load(&self) -> HashMap<Kind, VecDeque<u64>> {
 		File::open(&self.0)
-			.map_err(|e| Box::new(bincode::ErrorKind::IoError(e)))
-			.and_then(|mut file| bincode::deserialize_from(&mut file, bincode::Infinite))
+			.map_err(|e| Box::new(bincode::ErrorKind::Io(e)))
+			.and_then(|mut file| bincode::deserialize_from(&mut file))
```

Comment:
So this is implicitly unbounded now?

Reply:
yes, https://docs.rs/bincode/1.1.4/bincode/fn.config.html